content/rs/technology-behind-redis-enterprise.md

```diff
@@ -1,102 +1,73 @@
+
+You can read more about these features in our [architecture guide]({{< relref "/rs/concepts/_index.md" >}}).
 
-## Simplified and automated operations
+## Operational simplicity
 
-RS boosts the productivity of developers and DevOps by simplifying and
-automating certain complex and time-consuming ops-related tasks, including
-provisioning new databases, updating database configuration,
-resharding existing databases and rebalancing shards across cluster
+Redis Enteprise simplifies and automates many tasks including provisioning new databases, updating database configuration,
+resharding existing databases, and rebalancing shards across cluster
 nodes.
 
-The homogeneous, symmetric cluster architecture of RS enables the underlying
-infrastructure to be fully heterogeneous, where each node in the cluster
-can be physical or virtual, with variations in RAM capacity, storage
-devices, and the number of compute cores. This freedom of configuration
-allows the operator to roll out clusters that make the most efficient
-use of the resources, with minimal system limitations. For example, a
-cluster may be based on a few permanent bare-metal servers with
-additional virtual servers for scaling out (or in) when load increases
-or decreases. Built-in configurable monitoring functions help the
-administrator keep an eye on the resource utilization levels in the cluster.
-
-RS also makes sure that there is uninterrupted operation of Redis databases by protecting
-against out-of-memory events and tuning memory fragmentation
-automatically.
-
-All of the above is complemented by an easy-to-use web-based user
-interface, CLI tools, and APIs. Special experience or knowledge is not
-required to create, manage and monitor Redis clusters, nodes, and
-databases in RS.
-
-You can read more about the architecture of RS in [Concepts and
-Architecture]({{< relref "/rs/concepts/_index.md" >}}).
+The symmetric, shared-nothing cluster architecture allows for a variety
+of underlying infrastructures. Each cluster node can be physical or virtual, with variations in memory, storage devices, and number of compute cores. This freedom of configuration allows you to roll out clusters that most efficiently use your resources.
+
+For example, a cluster may be based on a few permanent bare-metal servers with
+additional virtual servers for scaling out when load increases
+or decreases.
+
+### Monitoring and alerting
+
+Monitoring and alerting are built-in and configurable. You can use the included dashboards to monitor your cluster, and you can easily [integrate with Nagios, Prometheus, and Grafana]({{< relref "/rs/administering/monitoring-metrics/_index.md" >}}).
+
+
+### UI, APIs, and CLIs
+
+You can administer a Redis Enterprise Software Cluster using the web-based UI,
+the cluster REST API, and various CLI-based tools. Once the cluster is up and running, you don't need any special experience or knowledge to create, manage, and monitor your databases and cluster nodes.
```

Comment:
Well it's not as easy as a simple microwave but I understand that the point is that the actions are simple.

Reply:
That text was there before. I'm removing it. Agreed.